src/uwtools/drivers/standalonedriver.py

```diff
@@ -0,0 +1,186 @@
+"""
+An abstract class for component drivers.
+"""
+
+import os
+import re
+import stat
+from datetime import datetime, timedelta
+from pathlib import Path
+from textwrap import dedent
+from typing import Any, Dict, List, Optional, Union
+
+from iotaa import asset, task, tasks
+
+from uwtools.drivers.driver import Driver
+from uwtools.exceptions import UWConfigError
+from uwtools.scheduler import JobScheduler
+from uwtools.utils.processing import execute
+
+
+class StandaloneDriver(Driver):
```

Comment:
I don't immediately see a reason why this can't live with `Driver` in `driver.py`, as both are closely-related base classes. I don't necessarily see why this one needs its own module. I suppose one could say the same for _all_ the driver modules, though I see a benefit to keeping imports tailored somewhat for each driver (which, then again, is also true here, as you've split the imports nicely). I don't know. Maybe it's just the slightly unwieldy name of this module... I think it would be fine to see
```
from uwtools.drivers.driver import Driver
```
or
```
from uwtools.drivers.drivers import StandaloneDriver
```
Just a thought. I see pros and cons.